lib/pavilion/test_config/file_format.py

```diff
@@ -277,6 +277,14 @@ class TestConfigLoader(yc.YamlConfigLoader):
                     'cmds', sub_elem=yc.StrElem(),
                     help_text='The sequence of commands to run to perform '
                               'the build.'),
+                yc.ListElem(
+                    'cmds_extend_before', sub_elem=yc.StrElem(),
+                    help_text='Commands to run before inherited build '
+                              'commands.'),
+                yc.ListElem(
+                    'cmds_extend_after', sub_elem=yc.StrElem(),
+                    help_text='Commands to run after inherited build '
+                              'commands.'),
```

Comment:
It just struck me that the names are terrible.

Let's do `prepend_cmds` and `append_cmds`.